trade-finance-manager-ui/server/controllers/case/index.test.js

```diff
@@ -147,7 +149,7 @@ describe('controllers - case', () => {
 
       describe('when deal cancellation is disabled', () => {
         it('should render the template with showDealCancelButton=false', async () => {
-          jest.mocked(isDealCancellationEnabled).mockReturnValueOnce(false);
+          jest.mocked(canDealBeCancelled).mockReturnValueOnce(false);
```

Comment:
Also, unless i'm missing it, could we add a test for "deal cancellation is permitted, but the deal cancellation status is _not_ `TFM_DEAL_CANCELLATION_STATUS.DRAFT` - for the `hasDraftCancellation` page/nunjucks variable

The alternative would be to extract the `hasDraftCancellation` const into it's own helper function and then that could just be called in the test once

Reply:
Great spot, these tests needed refactoring after updating the logic in the index file (explained why I had a failing test😅). Have now rewritten to reflect the draft cancellation status check